spec/pdc/v1/release_spec.rb

```diff
@@ -23,6 +23,11 @@
       count = release.where(active: false).count
       count.must_equal 0
     end
+
+    it 'can get release by multi product_version ' do
```

Comment:
Yeah honestly, with the nature of these tests it's hard to tell if it's doing the right thing. Seems alright.

Reply:
Yes, I think basically it can test whether the url is right or not. 
Thanks for reviewing that. :)